src/Pages/sangyeonkim/Login/Login.js

```diff
@@ -1,14 +1,113 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import "./Login.scss";
+import { FaFacebookSquare } from "react-icons/fa";
 
 class Loginkim extends Component {
-	state = {  }
-	render() { 
-		return (
-			<div>
-
-			</div>
-		);
-	}
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      pw: "",
+    };
+  }
+
+  goToMain = (e) => {
+    e.preventDefault();
+    if (this.state.id.includes("@") && this.state.pw.length > 4) {
+      fetch("http://10.58.4.128:8000/user/signin", {
+        method: "POST",
+        body: JSON.stringify({
+          email: this.state.id,
+          password: this.state.pw,
+        }),
+      })
+        .then((response) => response.json())
+        .then((result) => console.log("결과: ", result));
+      this.props.history.push("./Main-sangyeonkim");
+    }
+  };
+
+  handleIdInput = (event) => {
+    this.setState({ id: event.target.value });
+  };
+
+  handlePwInput = (event) => {
+    this.setState({ pw: event.target.value });
+  };
```

Comment:
같은 기능을 하는 함수네요!
**계산된 속성명** 이라는 키워드로 검색하시면 자료를 많이 찾을 수 있을 거에요!
```suggestion
  handleIdInput = (event) => {
    this.setState({ id: event.target.value });
  };

  handlePwInput = (e) => {
    this.setState({ [e.target.name]: e.target.value });
  };
```